server/repository/src/db_diesel/document_registry.rs

```diff
@@ -99,32 +101,33 @@ impl<'a> DocumentRegistryRepository<'a> {
 
         result
     }
+
+    pub fn create_filtered_query(filter: Option<DocumentRegistryFilter>) -> BoxedDocRegistryQuery {
```

Comment:
FWIW this doesn't use the connection in `Self` ('cause it returns a boxed query to use later), so doesn't really need to be associated with the struct. But it is elsewhere so I just did the same here